sunpy/coordinates/transformations.py

```diff
@@ -20,7 +20,7 @@
 from .representation import (SphericalWrap180Representation,
                              UnitSphericalWrap180Representation)
 from .frames import (HeliographicStonyhurst, HeliographicCarrington,
-                     Heliocentric, Helioprojective)
+                     Heliocentric, Helioprojective, HelioprojectiveRadial)
```

Comment:
I think either this needs to be HelioProjectiveRadial, or the class name needs to change in frames.py